.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
I am terribly uneducated here. Can you describe this file, how it is used and what each of lines represents?

Thanks!